.github/workflows/build-test.yml

```diff
@@ -34,6 +34,7 @@ jobs:
           key: yarn-cache-${{ runner.os }}-${{ steps.yarn-version.outputs.YARN_VERSION }}-${{ hashFiles('yarn.lock') }}-${{ matrix.node-version }}
       - name: Install Yarn dependencies
         run: yarn --immutable
+
```

Comment:
Since this file has grown a lot, I've added some spaces between jobs to make it more readable.